vmdb/spec/models/ems_refresh/refreshers/openstack_infra_refresher_rhos_juno_spec.rb

```diff
@@ -0,0 +1,113 @@
+
+require "spec_helper"
+
+describe EmsRefresh::Refreshers::OpenstackInfraRefresher do
+  before(:each) do
+    _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+    @ems = FactoryGirl.create(:ems_openstack_infra, :zone => zone, :hostname => "192.0.2.1",
+                              :ipaddress => "192.0.2.1", :port => 5000)
+    @ems.update_authentication(
+        :default => {:userid => "admin", :password => "22ffdb40bb99703b3650a79c1c1305a65d83e898"})
```

Comment:
We'll want to scrub these values.  Basically, in all the other vcr-based tests we scrub the ip, userid, and password.  It's a little annoying, but it's done so infrequently that no one has seen the need to script it away.

But, it means that you have to scrub the values here, and scrub them in the actual VCR recording, too.  Usually a simple `sed` command line will do.